src/FrameRipper.py

```diff
@@ -0,0 +1,111 @@
+import cv2
+import numpy as np
+import os
+import random
+
+# Frame Ripper Script WIP
+# Working on adding subset functionality in order to randomly grab sets of frames instead of all of them
+
+__author__ = 'Steve'
+
+# Function to scan the video files, processes and adds them to Array video_list
+def get_video_files(video_path = 'Videos'):
+    
+    entries = os.listdir(video_path)
+    video_list = []
+    
+    for entry in entries:
+        video_list.append(video_path + "/" + entry)
+    return video_list
+
+# Function that takes each element from the video_list and returns the total frame count of each video file
+def get_total_frames(videos = get_video_files()):
+    
+    total_frames = []
+    
+    for index, file_name in enumerate(videos):
+        cap = cv2.VideoCapture(file_name)
+        frame_count = int(cap.get(cv2.CAP_PROP_FRAME_COUNT))
+        total_frames.append(frame_count)
+    return total_frames
+
+# Video Processing
```

Comment:
we can see by function name that we are processing a video, don't need comments just to have comments.

Reply:
also this is only processing a video frame not the entire video (I think i did say process video tho, but it is only a frame)